### ISSUE TITLE
Decide on monitoring system

### ISSUE THREAD

**Issue by Thomsen96** [DevelOpsITU/MiniTwit#99]
Prometheous, Promtail, Nodeexporter, fluentbit, etc

**Comment by Turboen** [comment]
Decided prometheus, but needs to elaborate more before this can be closed.

**Comment by Zavir** [comment]
Initially we looked at the solutions proposed by the lecturer:

- [Nagios](https://www.nagios.com/products/nagios-log-server/)
- [Munin](http://munin-monitoring.org/)
- [Netdata](https://my-netdata.io/)
- [Shinken](http://www.shinken-monitoring.org/)
- [StatsD](https://github.com/statsd/statsd)
- [Prometheus](https://prometheus.io/)

Netdata and Prometheus seemed to be the most interesting, since they had the most comprehensive GitHub repositories aswell as **Go language support** that allows creation of custom metrics. However, Netdata seems like a "monitoring/visualization all-in-one solution" and we were more interested in having the visualization part as an independent process that received metrics from some monitoring service. Thus, we decided to use Prometheus and Grafana, since this worked perfectly in regard to our specifications. Since the exercises for session 06 were based on using Grafana/Prometheus it also came quite natural to use this for our application. 

Furthermore, some group members also had prior experience with Prometheus (and is actively using it in other courses). It also seemed the easiest/fastest to implement on our application server, which is something we value a lot, since the course is very heavy in regard to the workload. 

**TL;DR**:
- Previous experience with Prometheous made it easy to use.
- Prometheous is heavily reviewed in the exercises, making it easier to use/understand.
- Using Promethous in multiple different courses (two birds with one stone)
- Promethous had the most extensive documentation/GitHub repo
- etc..

**Comment by Lucki2g** [comment]
[DataDog ](https://www.datadoghq.com/) is a paid option, but we don't want to pay, when there is no need.
